static/app/views/explore/components/seerComboBox/seerSearchPopover.tsx

```diff
@@ -0,0 +1,54 @@
+import {useRef} from 'react';
+import styled from '@emotion/styled';
+import type {AriaPopoverProps} from '@react-aria/overlays';
+import {Overlay, usePopover} from '@react-aria/overlays';
+import type {OverlayTriggerState} from '@react-stately/overlays';
+
+interface PopoverProps extends Omit<AriaPopoverProps, 'popoverRef'> {
+  children: React.ReactNode;
+  containerRef: React.RefObject<HTMLDivElement | null>;
+  state: OverlayTriggerState;
+  popoverRef?: React.RefObject<HTMLDivElement | null>;
+}
+
+export function SeerSearchPopover(props: PopoverProps) {
+  const ref = useRef<HTMLDivElement>(null);
+  const {popoverRef = ref, state, children} = props;
+
+  const {popoverProps, underlayProps} = usePopover(
```

Comment:
We usually use `useOverlay`, I'm not sure if there are any uses of usePopver so probably bets to keep it consistent